apps/data/test/lib/projects_test.exs

```diff
@@ -30,4 +30,14 @@ defmodule Data.ProjectsTest do
     new_url = "https://example.com/new"
     assert {:ok, %{id: _id, url: ^new_url}} = Projects.update(project, %{url: new_url})
   end
+
+  test "can get all projects, paginated" do
+    Enum.each((1..2), fn(_) -> insert(:project) end)
```

Comment:
This can be refactored to: `insert_pair(:project)`.

For more than 2, we can use: `insert_list(_N_, :project)`